src/main/resources/org/jenkinsci/plugins/pipeline/utility/steps/maven/ReadMavenPomStep/help.html

```diff
@@ -28,4 +28,12 @@
     file.
     The returned object is a
     <a href="http://maven.apache.org/components/ref/3.3.9/maven-model/apidocs/org/apache/maven/model/Model.html">Model</a> .
-</p>
\ No newline at end of file
+</p>
+<p>
+    Avoid using this step and <code>writeMavenPom</code>.
+    It is better to use the <code>sh</code> step to run <code>mvn</code> goals.
+    For example:
+</p>
+<pre><code>
+def version = sh script: 'mvn help:evaluate -Dexpression=project.version -q -DforceStdout', returnStdout: true
```

Comment:
Credit to [this post](https://blog.soebes.de/blog/2018/06/09/help-plugin/) by @khmarbaise for the use of the `-q -DforceStdout` options, without which the output of `help:evaluate` is hard to parse.